cli/slsa-verifier/verify/options.go

```diff
@@ -127,6 +127,59 @@ func (o *VerifyNpmOptions) AddFlags(cmd *cobra.Command) {
 	cmd.MarkFlagsMutuallyExclusive("source-versioned-tag", "source-tag")
 }
 
+// VerifyVSAOptions is the top-level options for the `verifyVSA` command.
+type VerifyVSAOptions struct {
+	SubjectDigests    []string
+	AttestationsPath  string
+	VerifierID        string
+	ResourceURI       string
+	VerifiedLevels    []string
+	PublicKeyPath     string
+	PublicKeyID       string
+	PublicKeyHashAlgo string
+	PrintAttestation  bool
+}
+
+var _ Interface = (*VerifyVSAOptions)(nil)
+
+// AddFlags implements Interface.
+func (o *VerifyVSAOptions) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().StringArrayVar(&o.SubjectDigests, "subject-digest", []string{},
+		"the digests to be verified. Pass multiple digests by repeating the flag.")
```

Comment:
Do we need to document that format for each digest, that it's `alg:digest`?

Reply:
added